StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/ElementsCustomer.swift

```diff
@@ -40,9 +40,14 @@ struct ElementsCustomer: Equatable, Hashable {
         return ElementsCustomer(paymentMethods: paymentMethods, defaultPaymentMethod: defaultPaymentMethod, customerSession: customerSession)
     }
 
+    func getDefaultPaymentMethod() -> STPPaymentMethod? {
+        let defaultSavedPaymentMethod = paymentMethods.first { $0.stripeId == defaultPaymentMethod }
+        return defaultSavedPaymentMethod
+    }
+
     func getDefaultOrFirstPaymentMethod() -> STPPaymentMethod? {
```

Comment:
in PaymentSheetVerticalViewController, we have this logic for the initialSelection
```
                    if let customer = elementsSession.customer,
                       let defaultPaymentMethod = customer.getDefaultOrFirstPaymentMethod() {
                        customerDefault = CustomerPaymentOption.stripeId(defaultPaymentMethod.stripeId)
                    }
```
This seems to indicate that the first payment method can become the default. Is this true?  Ultimately, i'm trying to wrap my head around when we would access defaultPaymentMethod vs defaultOrFirst.

Reply:
From my understanding, the selected payment method should be the default if it exists or the first payment method in the customer's list of saved payment methods. However, the default badge itself should only be displayed if there is a set default in the elementsSession customer.

So we access defaultOrFirst when trying to determine the selected payment method, while we access defaultPaymentMethod to determine if the default badge should be shown and which payment method to show it on.

Reply:
Ah thanks for the explanation!